project/deps.sc

```diff
@@ -47,7 +47,7 @@ object Scala {
   }
 
   def maxAmmoniteScala212Version = scala212
-  def maxAmmoniteScala213Version = scala213
+  def maxAmmoniteScala213Version = "2.13.13"
```

Comment:
Btw. There is already support https://repo1.maven.org/maven2/com/lihaoyi/ammonite_2.13.14/

Reply:
I'll bump it in the other PR